calyx/src/ir/printer.rs

```diff
@@ -522,4 +522,13 @@ impl Printer {
             ),
         }
     }
+
+    /// Formats the top-level metadata if present
+    pub fn format_metadata(metadata: &Option<String>) -> String {
+        if let Some(metadata_str) = metadata {
+            format!("METADATA #{{\n{}\n}}#\n", metadata_str)
```

Comment:
dumb nit, can we make this `metadata` instead? All other keywords are small caps.

Reply:
technically the metadata label is case insensitive, so we can change the default printing pretty easily

Reply:
lower-cased!